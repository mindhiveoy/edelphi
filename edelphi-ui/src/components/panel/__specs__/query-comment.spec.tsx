import React from "react";
import { shallow } from "enzyme";
import { QueryCommentClass } from "../query-comment";

describe("QueryComment", () => {
  it("Renders the comment", () => {
    const queryComment = shallow(
      <QueryCommentClass
        locale="fi"
        panelId={1}
        queryId={2}
        pageId={3}
        queryReplyId={4}
        canManageComments
        category={null}
        comment={{
          queryPageId: 1,
          queryReplyId: 2,
          contents: "Ei pidÃ¤ paikkaansa"
        }}
      />
    );
    expect(queryComment).toMatchSnapshot();
  });
});
