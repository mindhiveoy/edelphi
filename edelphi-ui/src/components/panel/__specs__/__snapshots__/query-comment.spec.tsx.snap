// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`QueryComment Renders the comment 1`] = `ShallowWrapper {}`;
